README.md

```diff
@@ -2,40 +2,64 @@
 
 ## Summary
 
-`postgresql-transactional` is a simple monadic wrapper around the SQL primitives introduced by the [postgresql-simple][psqls] package. It provides simple and predictable semantics for database operations, enforces awareness of Postgres's transactional nature at API boundaries, and obviates the need for transaction boilerplate in SQL queries.
+`postgresql-transactional` is a simple monadic wrapper around the SQL
+primitives introduced by the [postgresql-simple][psqls] package. It provides
+simple and predictable semantics for database operations, enforces awareness of
+Postgres's transactional nature at API boundaries, and obviates the need for
+transaction boilerplate in SQL queries.
 
 ## Details
 
-Though the primitives provided by the [postgresql-simple][psqls] package are fast and powerful, their interface is (by design) very basic: specifically, all query functions take a shared `Connection` parameter and operate in the `IO` monad. 
+Though the primitives provided by the [postgresql-simple][psqls] package are
+fast and powerful, their interface is (by design) very basic: specifically, all
+query functions take a shared `Connection` parameter and operate in the `IO`
+monad. 
 
 ```haskell
 query :: FromRow r => Connection -> Query -> IO [r]
 execute :: ToRow q => Connection -> Query -> q -> IO Int64
 ```
 
-By virtue of the fact that (usually) all queries in a given scope are routed through a single `Connection`, we can abstract away the shared `Connection` parameter by wrapping a `ReaderT Connection` around the `IO` monad:
+By virtue of the fact that (usually) all queries in a given scope are routed
```

Comment:
Does this have manual line breaks now, or is the whitespace in this diff just being dodgy?